.github/workflows/rust.yaml

```diff
@@ -50,11 +50,4 @@ jobs:
         if: steps.changes.outputs.rust == 'true'
         run: nix-shell --run "./scripts/rust/linter.sh"
       - name: Build rust binaries
-        if: steps.changes.outputs.rust == 'true'
-        run: nix-shell --run "./scripts/rust/builder.sh"
-      - name: Ensure Cargo.lock is up to date
-        run: |
-          if git diff --name-only | grep -q 'Cargo.lock'; then
-            echo "Cargo.lock is not up to date!"
-            exit 1
-          fi
+        run: nix-shell --run "cargo build --bins"
```

Comment:
not sure why you're modifying this?

Reply:
Ohh, ill keep Ensure Cargo.lock up to date. We wont run the build.sh as we have Cargo.toml now. `nix-shell --run "cargo build --bins"` was there previously also.

Reply:
Does this work without the cd?

Reply:
Yes it runs on root.